content/ch/author/benjamin-bengfort.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Benjamin Bengfort"
+image: ""
+email: "benjamin@rotational.io"
+social:
+  - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+    link : "https://www.linkedin.com/in/bbengfort/"
+  - icon : "ti-twitter-alt" # themify icon pack : https://themify.me/themify-icons
+    link : "https://twitter.com/bbengfort"
+  - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+    link : "https://github.com/bbengfort"
+---
+
+Founder and CEO, Rotational Labs, LLC.
+
+Creator of Yellowbrick, machine learning practitioner, distributed systems developer, data science educator, O’Reilly author, Guyanese-American, BBQ enthusiast
```

Comment:
@KNGJOEL I think we can remove this file in favor of adopting the go-18n approach